src/JK/WebsiteBundle/Controller/IndexController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace JK\WebsiteBundle\Controller;
+
+use JK\CommonBundle\Controller\AbstractController;
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class IndexController extends AbstractController
+{
+    private $templating;
```

Comment:
Template engine is common dependency for all controllers, so you have to move it to `AbstractController`.